packages/dataviews/src/types.ts

```diff
@@ -7,6 +7,7 @@ import type { ReactElement, ComponentType } from 'react';
  * Internal dependencies
  */
 import type { SetSelection } from './private-types';
+import type { ModalProps } from '../../components/src/modal/types';
```

Comment:
We should not reference code in another package via relative paths. If really wanted to use the props type of the `Modal` component, the recommended way should be something like the following:

```
import { Modal } from `@wordpress/components`;

// ...

type ModalProps = React.ComponentProps< typeof Modal >;
```